.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -33,50 +33,19 @@ body:
 
 - type: dropdown
   attributes:
-    label: MSAL client type
+    label: Scenario
     description: "Are you using PublicClientApplication,  ConfidentialClientApplication or ManagedIdentityApplication?"
     multiple: true
     options: 
-      - "Public"
-      - "Confidential"
-      - "Managed identity"
+      - "PublicClient - desktop app"
+      - "PublicClient - mobile app"
+      - "ConfidentialClient - web site (AcquireTokenByAuthCode)"
```

Comment:
nit: Isn't it possible for someone to have two of the CCA scenarios? perhaps it should be one generic option since we would ultimately treat them the same?